tests/test_env.py

```diff
@@ -0,0 +1,67 @@
+import runhouse as rh
+
+
+pip_reqs = ["torch", "numpy"]
+
+
+def test_create_env_from_name_local():
+    env_name = "~/local_env"
+    local_env = rh.env(name=env_name, reqs=pip_reqs).save()
+    del local_env
+
+    remote_env = rh.env(name=env_name)
+    assert remote_env.reqs == pip_reqs
+
+
+def test_create_env_from_name_rns():
+    env_name = "rns_env"
+    env = rh.env(name=env_name, reqs=pip_reqs).save()
+    del env
+
+    remote_env = rh.env(name=env_name)
+    assert remote_env.reqs == pip_reqs
+
+
+def test_create_env():
+    test_env = rh.env(name="test_env", reqs=pip_reqs)
+    assert len(test_env.reqs) == 2
+    assert test_env.reqs == pip_reqs
+
+
+def test_to_system():
+    test_env = rh.env(name="test_env", reqs=["numpy"])
+    system = rh.cluster("^rh-cpu").up_if_not()
+
+    test_env.to(system)
+    res = system.run_python(["import numpy"])
```

Comment:
How confident are you that this wouldn't work before you send over the env?😄 Also other tests we run may install it first. Maybe better to do a super random library?

Reply:
ok, switching to diffusers but lmk if you have another python package in mind